src/main/resources/init_db.sql

```diff
@@ -0,0 +1,10 @@
+CREATE DATABASE 'homework';
+
+USE `homework`;
```

Comment:
you shouldn't create whole db in init script

you should rather create a schema and use it - if you wish

otherwise you can just create a table